content/docs/user-guide/scripting/lua/_index.md

```diff
@@ -6,18 +6,24 @@ toc: true
 weight: 100
 ---
 
-**Topics**
-+ [Adding Lua Scripts to Component Entities](/docs/user-guide/editor/add-lua-script)
-+ [Properties Table](properties/)
-+ [Using EBuses in Lua](ebus/)
-+ [Lua Editor](lua-editor/)
-+ [Debugging Lua Scripts](debugging-scripts/)
-+ [The Lua Environment (Advanced)](environment/)
-
-You can use Lua in O3DE to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
+You can use Lua in **Open 3D Engine (O3DE)** to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
 
 O3DE uses Lua version {{< versions/lua >}}.
 
+## Building Lua Editor and GridHub
+
+**Lua Editor** (Lua IDE) and GridHub, which is O3DE's central connection hub for debugging, must be built for every project that requires them. Include `LuaIDE` and `GridHub` as build targets when you build your project: `--target <project>.GameLauncher Editor LuaIDE GridHub`.  
+
+For information about accessing class references in Lua Editor, refer to [Debugging with Lua Editor](debugging-tutorial).
+
+### Example
+
+If you need to build Lua Editor and GridHub after your project is built, navigate to your O3DE source directory, for example `c:\o3de`, and run the following command, inserting the path to your project's build directory.
+
+```
+cmake --build c:\<o3de-projects>\<project>\build --target LuaIDE GridHub --config profile -- /m
```

Comment:
Build directory is configurable, not fixed, so let's put that in angle brackets too.
```suggestion
cmake --build c:\<o3de-projects>\<project>\<build> --target LuaIDE GridHub --config profile -- /m
```